grails-app/views/inventory/browse.gsp

```diff
@@ -151,7 +151,7 @@
 
 								</form>
 							</div>
-							<g:if test="${commandInstance?.searchResults instanceof grails.orm.PagedResultList}">
+							<g:if test="${commandInstance?.searchResults instanceof PagedResultList}">
```

Comment:
Could you try using `grails.gorm.PagedResultList` (**g**orm)? (replace the `org.pih.warehouse.PagedResultList` with it in the `searchProducts` method)

Reply:
I would agree with this ... if it was easy. Unfortunately, I remember the PagedResultList class (there seem to be two in Grails 3: grails.orm.PagedResultList and grails.gorm.PagedResultList) does not have a constructor that supports our use case. So if there's not an easy way to do it, I would leave it alone and create a ticket to refactor this by fixing the query itself so we're returning the correct paginated data from the query.

Reply:
Yup, you're right @jmiranda. There is no constructor that supports our use case: 
`Could not find matching constructor for: grails.orm.PagedResultList(java.util.ArrayList, java.lang.Integer)`